packages/driver/cypress/integration/commands/navigation_spec.js

```diff
@@ -85,7 +85,7 @@ describe('src/cy/commands/navigation', () => {
     })
 
     // TODO: fix this
-    it.skip('(FLAKY) sets timeout to Cypress.config(pageLoadTimeout)', {
+    it('(FLAKY) sets timeout to Cypress.config(pageLoadTimeout)', {
```

Comment:
With retries setup, are these passing reliably now?
```suggestion
    it('sets timeout to Cypress.config(pageLoadTimeout)', {
```